settings.php

```diff
@@ -48,6 +48,26 @@
     );
     $settings->add(new admin_setting_configselect($name, $title, $description, $default, $choices));
 
+    /* Toggle display block choices */
+    $name = 'format_topcoll/defaultdisplayblocks';
+    $title = get_string('defaultdisplayblocks', 'format_topcoll');
+    $description = get_string('defaultdisplayblocks_desc', 'format_topcoll');
+    $default = array('0,1,2,3');
+    $choices = array('search_forums','news_items','calendar_upcoming','recent_activity');
```

Comment:
Not quite!  Use an associative array -> array(
'search_forums' => get_string(...),
'news_items' => get_string(...),
'calendar_upcoming' => get_string(...),
'recent_activity' => get_string(...)
)

then need to work out how to set the default.

Reply:
Also this list needs to be generated from the list of blocks installed.  As the above are 'core' blocks then should not be an issue with the default.